crates/cli/src/error.rs

```diff
@@ -217,35 +217,59 @@ struct ErrorFormat<'a> {
 }
 
 impl<'a> fmt::Display for ErrorFormat<'a> {
-  fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+  fn fmt(&self, _: &mut fmt::Formatter<'_>) -> fmt::Result {
     let ErrorMessage {
       title,
       description,
       link,
     } = ErrorMessage::from_context(self.context);
-    let bold = Style::new().bold();
-    let error = Color::Red.paint("Error:");
-    let message = bold.paint(title);
-    writeln!(f, "{error} {message}")?;
-    let help = Color::Blue.paint("Help:");
-    writeln!(f, "{help} {description}")?;
+
+    write_with_style(
+      "Error: ",
+      Some(ColorSpec::new().set_fg(Some(Color::Red)).to_owned()),
+    );
+    write_with_style(
+      format!("{title}\n"),
+      Some(ColorSpec::new().set_bold(true).to_owned()),
+    );
+    write_with_style(
+      "Help: ",
+      Some(ColorSpec::new().set_fg(Some(Color::Blue)).to_owned()),
```

Comment:
Can we add a helper method to transform Color to ColorSpec?

Reply:
Do you mean something like:

```rs
pub fn to_spec(color: Color) -> ColorSpec {
  ColorSpec::new().set_fg(Some(color)).to_owned()
}
```

And then we can simplify the write call:

```rs
write_with_style("Error: ", Some(to_spec(Color::Red)));
```

Reply:
Yup. I would wrap it with Some as well.